src/foam/nanos/logger/StdoutLogger.java

```diff
@@ -52,6 +52,7 @@ public String format(LogRecord record) {
         str.append(record.getLevel());
       }
 
+      str.append(" ");
```

Comment:
Why is it a space instead of a comma?

Reply:
I can change it to a comma (or would it be better to change it to a space followed by a comma?)

Reply:
To be consistent with the comma after the timestamp I think it should just be a comma and no space.